src/pages/Product/Body/Comment/CommentCard/CommentCard.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+// import './Body.scss';
+// import imgA from '../../../assets/images/backimg1.jpg';
+// import imgB from '../../../../assets/images/backimg2.jpg';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faThumbsUp } from '@fortawesome/free-solid-svg-icons';
+import { faComment } from '@fortawesome/free-solid-svg-icons';
+// import { faChevronCircleLeft } from '@fortawesome/free-solid-svg-icons';
+// import { faChevronCircleRight } from '@fortawesome/free-solid-svg-icons';
+
+// import { ReactComponent as Star } from '../../../assets/images/star.svg';
+// import { ReactComponent as StarOn } from '../../../assets/images/starbrown.svg';
+
+class CommentCard extends React.Component {
+  render() {
+    const { url, id, stars, comment, thumb, talk } = this.props;
```

Comment:
```suggestion
    const { url, id, stars, comment, thumb, talk } = this.props.commentList;
```
이전 리뷰에 따라 구조분해할 때 이런식으로 변경해주면 되겠네요 :)